ludicrousdb/includes/class-ludicrousdb.php

```diff
@@ -735,8 +737,8 @@ public function db_connect( $query = '' ) {
 							$this->current_host         = $host_and_port;
 							$this->dbh2host[ $dbhname ] = $host_and_port;
 
-							// Define these to avoid undefined variable notices
-							$queries = isset( $queries   ) ? $queries   : 1;
+							// Define these to avoid undefined variable notices.
+							$queries = isset( $queries ) ? $queries : 1;
```

Comment:
Column alignment needs to be allowed for things like this. Or they should be made into multiline ternaries so they can be understood more quickly.